Hosting/NetCord.Hosting.Services/Commands/CommandHandler.cs

```diff
@@ -46,7 +46,7 @@ internal unsafe partial class CommandHandler<TContext> : IGatewayEventHandler<Me
 
         _getPrefixLengthAsync = GetGetPrefixLengthAsyncDelegate(optionsValue);
         _createContext = optionsValue.CreateContext ?? ContextHelper.CreateContextDelegate<Message, GatewayClient, TContext>();
-        _handleResultAsync = optionsValue.HandleResultAsync;
+        _handleResultAsync = optionsValue.ResultHandler.HandleResultAsync;
```

Comment:
I think storing the handler and executing the method on it directly would be better